src/containers/AccountConnection.jsx

```diff
@@ -257,47 +265,45 @@ class AccountConnection extends Component {
                   }
                 </div>
             }
-            <AccountLoginForm
-              t={t}
-              connectorSlug={connector.slug}
-              isOAuth={connector.oauth}
-              fields={fields}
-              submitting={submitting}
-              deleting={deleting}
-              values={existingAccount ? existingAccount.auth || existingAccount.oauth : {}}
-              error={credentialsError}
-              forceEnabled={!!error}
-              onDelete={() => this.deleteAccount()}
-              onSubmit={(values) => this.submit(Object.assign(values, {folderPath: t('account.config.default_folder', connector)}))}
-              onCancel={() => this.cancel()}
-            />
-          </div>
-          <div className={styles['col-account-connection-data']}>
-            { hasDescriptions && hasDescriptions.service &&
-              <div>
-                <h4>{t('account.config.data.service.description')}</h4>
+            {isSuccessTimedOut
+              ? <div className={styles['col-account-connection-successTimeout']}>
```

Comment:
Maybe we could use `col-account-connection-successTimedOut` to keep coherent with other naming, what do you think?